components/Inputs.js

```diff
@@ -40,6 +42,12 @@ export default function InputsOne({
       <StyledVariatonButton type="button" onClick={() => handleVariation()}>
         Variation
       </StyledVariatonButton>
```

Comment:
Just a reminder:
You have a little typo there "StyledVariatonButton" instead of "VariationButton". If it was supposed to be as it is, than forget my writing :-)

Reply:
Thank you Jana, i changed the name :)